src/main/java/io/zold/api/RtTransaction.java

```diff
@@ -119,10 +150,10 @@ public int id() throws IOException {
     public ZonedDateTime time() throws IOException {
         return new ZonedDateTimeOf(
             new UncheckedText(
-                new IoCheckedScalar<>(
-                    new ItemAt<>(
-                        1, new SplitText(this.transaction, ";")
-                    )
+                new IoCheckedScalar<String>(
+                    () -> new ItemAt<Text>(
```

Comment:
@paulodamaso ditto, we can avoid the lambda here and re-wrapping `String` by changing `transaction` to `IoCheckedScalar`.